script/binary

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -e
+
+if [ -z "$1" ]; then
+    OS_PLATFORM_ARG=(-os="darwin linux windows")
+else
+    OS_PLATFORM_ARG=($1)
+fi
+
+if [ -z "$2" ]; then
+    OS_ARCH_ARG=(-arch="386 amd64 arm")
+else
+    OS_ARCH_ARG=($2)
+fi
+
+# Get rid of existing binaries
+rm -f docker-compose*
+rm -rf Godeps/_workspace/pkg
+
+# Build binaries
+gox "${OS_PLATFORM_ARG[@]}" "${OS_ARCH_ARG[@]}" \
```

Comment:
Because that's what was used before (https://github.com/docker/libcompose/pull/11/files#diff-1293c4cc4b2f3a632b921bd168da9a8eL24). the original reason -> @ibuildthecloud :stuck_out_tongue_closed_eyes: